flow.cylc

```diff
@@ -261,165 +261,83 @@ combine-statics => clean-pp-statics
 {% endif %}
 """
 
+{% set PP_CHUNKS = [PP_CHUNK_A] %}
 
-        #
-        # Recurrence interval: every CHUNK-A
-        #
-        {# Run the tasks to process CHUNK-A every CHUNK-A, starting after PP_CHUNK_A #}
-        +{{ PP_CHUNK_A | subtract_durations('P1Y') }}/{{ PP_CHUNK_A }} = """
-
-{# Tasks can be split over multiple lines if the subsequent ones begin with =>, &, or |                   #}
-{# The make-timeseries tasks for CHUNK-A depend on all segment processing for the time period succeeding. #}
-{# This Jinja for loop expands to do this.                                                                #}
-{% if DO_REGRID %}
-rename-split-to-pp-regrid<regrid>
-    {% for SEGMENT in range(1, HIST_SEGMENTS_PER_CHUNK_A) %}
-& rename-split-to-pp-regrid<regrid>[{{ HISTORY_SEGMENT | multiply_duration(-SEGMENT) }}]
-    {% endfor %}
-    {% if HISTORY_SEGMENT != PP_CHUNK_A %}
-=> make-timeseries-regrid-{{ PP_CHUNK_A }}<regrid>
-    {% endif %}
-    {% if DO_TIMEAVGS %}
-=> make-timeavgs-regrid-{{ PP_CHUNK_A }}<regrid>
-    {% endif %}
-{% endif %}
-
-{% if DO_NATIVE %}
-rename-split-to-pp-native<native>
-    {% for SEGMENT in range(1, HIST_SEGMENTS_PER_CHUNK_A) %}
-& rename-split-to-pp-native<native>[{{ HISTORY_SEGMENT | multiply_duration(-SEGMENT) }}]
-    {% endfor %}
-    {% if HISTORY_SEGMENT != PP_CHUNK_A %}
-=> make-timeseries-native-{{ PP_CHUNK_A }}<native>
-    {% endif %}
-    {% if DO_TIMEAVGS %}
-=> make-timeavgs-native-{{ PP_CHUNK_A }}<native>
-    {% endif %}
-{% endif %}
-
-{% if DO_REGRID %}
-{{ "regrid-xy" | form_remap_dep('temporal', PP_CHUNK_A, PP_COMPONENTS, 'ts', HISTORY_SEGMENT) }}
-    {% if DO_TIMEAVGS %}
-{{ "regrid-xy" | form_remap_dep('temporal', PP_CHUNK_A, PP_COMPONENTS, 'av', HISTORY_SEGMENT) }}
-    {% endif %}
-{% endif %}
-
-{% if DO_NATIVE %}
-{{ "native" | form_remap_dep('temporal', PP_CHUNK_A, PP_COMPONENTS, 'ts', HISTORY_SEGMENT) }}
-    {% if DO_TIMEAVGS %}
-{{ "native" | form_remap_dep('temporal', PP_CHUNK_A, PP_COMPONENTS, 'av', HISTORY_SEGMENT) }}
-    {% endif %}
-{% endif %}
-
-{% if DO_TIMEAVGS %}
-remap-pp-components-av-{{ PP_CHUNK_A }}<component> => combine-timeavgs-{{ PP_CHUNK_A }}<component>
-    {% if CLEAN_WORK %}
-COMBINE-TIMEAVGS-{{ PP_CHUNK_A }}:succeed-all => clean-pp-timeavgs-{{ PP_CHUNK_A }}
-    {% endif %}
+{% if DO_SECONDARY_PP %}
+    {% do PP_CHUNKS.append(PP_CHUNK_B) %}
```

Comment:
Nice. Using "pp chunks" as a list is so much better than A and B only, thank you!

I think we have "chunk a" or "chunk b" used elsewhere, once or twice (analysis scripts, at least), but when that is undone, we should go with a list specified in the yaml instead of A and B.